### PR TITLE
Added yaml_edit

### PR DESCRIPTION
**Pull request by walnutdust**
Adding the yaml_edit package!

### CODE REVIEW

File: yaml_edit/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Yaml Editor
+
+A library for [YAML](https://yaml.org) manipulation while preserving comments.
+
```
Comment (jonasfj) [inline yaml_edit/README.md:4]: Can we add:
```markdown
**Disclaimer:** This is not an officially supported Google product.
```

File: yaml_edit/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+## v1.0.0
+
+- Initial Open Source Release
```
Comment (jonasfj) [inline yaml_edit/CHANGELOG.md:3]: Just `Initial release.` like the other CHANGELOGs is fine.

File: yaml_edit/analysis_options.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Defines a default set of lint rules enforced for
```
Comment (jonasfj) [inline yaml_edit/analysis_options.yaml:1]: Can we make this file just:
https://github.com/google/dart-neats/blob/73ada3d69f092cef445dd54e4e79953f8af3b969/safe_url_check/analysis_options.yaml

File: yaml_edit/example/example.dart [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import 'package:yaml_edit/yaml_edit.dart';
```
Comment (jonasfj) [inline yaml_edit/example/example.dart:1]: License header, example:
https://github.com/google/dart-neats/blob/73ada3d69f092cef445dd54e4e79953f8af3b969/safe_url_check/example/main.dart#L1-L13

This goes for all files.

File: yaml_edit/lib/src/editor.dart [hunk 5]
```diff
@@ -0,0 +1,632 @@
+// Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
```
Comment (jonasfj) [inline yaml_edit/lib/src/editor.dart:3]: License header: https://github.com/google/dart-neats/blob/73ada3d69f092cef445dd54e4e79953f8af3b969/safe_url_check/example/main.dart#L1-L13

Comment (jonasfj) [reply yaml_edit/lib/src/editor.dart:3]: Also feel free to update the date to 2020, and add your name as author... given that you're the original author :D

File: yaml_edit/test/test.dart [hunk 6]
```diff
@@ -0,0 +1,13 @@
+// BSD-style license that can be found in the LICENSE file.
```
Comment (jonasfj) [inline yaml_edit/test/test.dart:1]: Is this an accidentally add file?

Comment (walnutdust) [reply yaml_edit/test/test.dart:1]: whoops yeap - thanks for catching it!

File: yaml_edit/README.md (hunk 1 above)
Comment (walnutdust) [reply yaml_edit/README.md:4]: done!

**Pull request merged**